app/configuration/Config.scala

```diff
@@ -111,7 +114,7 @@ object Config {
         prpIds.digitalPackYearly
       )),
       campaignName = "DigiPack - £30 digital gift card",
-      codes = PromoCodeSet(PromoCode("DGA88"), PromoCode("DGB88")),
+      codes = PromoCodeSet(PromoCode("DGA88"), promoCodes:_*),
```

Comment:
http://stackoverflow.com/questions/6051302/what-does-colon-underscore-star-do-in-scala